imgui.ini

```diff
@@ -0,0 +1,24 @@
+[Window][Debug##Default]
```

Comment:
remove this lol. I apologize that GGUI keeps generating this... Maybe we could put it in a gitignore.